### ISSUE TITLE
Version 0.11.2 uses PHP 7 features and breaks BC with PHP < 7

### ISSUE THREAD

**Issue by sgehrig** [markuspoerschke/iCal#94]
The `Eluceo\iCal\Property\Event\Geo` class uses PHP 7 scalar type hints and therefore breaks BC with PHP < 7. It should be compatible with PHP >= 5.3.0 as stated in `composer.json`.

If PHP 7 is a strict requirement now, this version should be a new major version to follow semantic versioning IMHO.

```
Fatal Compile Error: Default value for parameters with a class type hint can only be NULL
```

**Comment by markuspoerschke** [comment]
Same as #91. I will create a tag now.

**Comment by markuspoerschke** [comment]
Thanks for reporting!

**Comment by markuspoerschke** [comment]
Should be fixed in `0.11.3`. PHP will become a requirement in the next Version `0.12.0`.
